HealthcareSystem/PatientWebApp/wwwroot/js/survey_result.js

```diff
@@ -58,33 +53,45 @@
 	});
 
 
-	$.ajax({
-		url: "/api/survey/surveyResultAboutDoctor/1308987105625",
-		type: "GET",
-		dataType: 'json',
-		processData: false,
-		contentType: false,
-		success: function (surveyResult) {
+	//***************************************************************************** on change  
+	/*
+	function changeDoctor(event) {
```

Comment:
Is there a specific reason why this was left in a comment when it's the same as the function starting at line 162 as far as I can see?

Reply:
I will delete it

Reply:
Solved in 4ed9a1ff7a23c5eb0e299a351efd936157d1020a.